doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -10,37 +10,40 @@ aliases:
   - /connect-sources/static-ips/
 ---
 
-Your Materialize region initiates connections from a static set of IP addresses.
-When connecting Materialize to services in your private networks (e.g., Kafka
-clusters or PostgreSQL databases), you must configure any firewalls to allow
-connections from these IP addresses.
-
-## Details
+## Static egress IPs
 
-Your Materialize region is associated with a static set of egress IP addresses.
-Most regions have four egress IP addresses, but this is not guaranteed. All
-connections to the public internet initiated by a source or sink in your region
-will originate from one of these egress IP addresses.
-
-We do not allocate unique IP addresses for each Materialize region. Multiple
-regions may share the same egress IP addresses.
+Each materialize region is associated with a unique set of of static egress IP
+addresses. All connections to the public internet initiated by a source or sink
+in your region will originate from an address in the provided ranges.
 
 When connecting Materialize to services in your private networks (e.g., Kafka
 clusters or PostgreSQL databases), you must configure any firewalls to allow
-connections from your region's egress IP addresses. You must allow connections
-from all egress IP addresses associated with your region. Connections may
-originate from any one of the egress IP addresses.
+connections from all CIDRs associated with your region. Connections may
+originate from any address in the region.
 
-To find the egress IP addresses associated with your region, you can query the
-[`mz_egress_ips`](/sql/system-catalog/mz_catalog/#mz_egress_ips) system table.
+Region          | CIDR
+----------------|------------
+`aws/us-east-1` | 98.80.4.128/27
+`aws/us-east-1` | 3.215.237.176/32
+`aws/us-west-2` | 44.242.185.160/27
+`aws/us-west-2` | 52.37.108.9/32
+`aws/eu-west-1` | 108.128.128.96/27
+`aws/eu-west-1` | 54.229.252.215/32
 
 {{< note >}}
-On rare occasion, we may need to change the egress IP addresses associated with
+On rare occasion, we may need to change the static egress IP addresses associated with
 a region. We make every effort to provide advance notice of such changes.
 {{< /note >}}
 
 
-## Example
+
+## Fetch static egress IPs
+You can fetch the egress IP addresses associated with your region by querying
+the [`mz_egress_ips`](/sql/system-catalog/mz_catalog/#mz_egress_ips) system
+table.
+
+
+### SQL Example
```

Comment:
```suggestion
```

Reply:
Could you remove the sentence below, and just have the example directly under the paragraph above?

Reply:
(I can also push all these adjustments in later, just on mobile now.)